packages/client/src/utils/authClient.ts

```diff
@@ -7,17 +7,21 @@ export type AuthClient = {
   signup: (credentials: SignUpCredentials) => Promise<User>;
 };
 
-export type Credentials = { email: string; password: string };
-export type SignUpCredentials = {
+export interface Credentials {
+  email: string;
+  password: string;
+}
+
+export interface SignUpCredentials {
   email: string;
   firstName: string;
   lastName: string;
   password: string;
-};
-export type User = {
+}
+export interface User {
```

Comment:
Yeah maybe it comes down to a habit but I tend to use interfaces to type an object because they are very close, you can extend or merge two interfaces etc..